source/Octopus.Tentacle/Configuration/Instances/ApplicationInstanceSelector.cs

```diff
@@ -121,12 +121,9 @@ AggregatedKeyValueStore ContributeAdditionalConfiguration(XmlFileKeyValueStore w
                     if (fileSystem.FileExists(rootPath))
                         return (null, rootPath);
 
-                    if (!applicationInstanceStore.TryLoadInstanceDetails(null, out var indexDefaultInstance))
-                    {
-                        throw new ControlledFailureException("There are no instances of OctopusServer configured on this machine. " +
-                            "Please run the setup wizard, configure an instance using the command-line interface or specify a configuration file");
-                    }
-
+                    // This will throw a ControlledFailureException if it can't find the instance so it won't be null
```

Comment:
Now that this is true, could we remove the "!" below (i.e. in `return (indexDefaultInstance!.InstanceName, indexDefaultInstance!.ConfigurationFilePath);`)

Reply:
I did consider removing it, but thought the suppression of the nullable warning might still be desirable with exception-based flow control. 
Happy to tidy it up to get rid of cruft though 👍🏽